pgsql-env

```diff
@@ -0,0 +1,14 @@
+POSTGRES_USER=lemur
+POSTGRES_PASSWORD=12345
```

Comment:
shall we leave a comment, to avoid false bug bounty reports?
for example:
# this DB password is used for testing in development environment. use strong passwords for your DB